webcomponents/highlight-code/src/components/highlight-code/deckdeckgo-highlight-code.tsx

```diff
@@ -212,9 +217,11 @@ export class DeckdeckgoHighlightCode {
         // if the language definition doesn't exist or if unpkg is down, display code anyway
         this.prismLanguageLoaded.emit(lang);
       };
+      const definition = deckdeckgoHighlightCodeLanguages[this.language];
 
-      script.src = 'https://unpkg.com/prismjs@latest/components/prism-' + lang + '.js';
-      script.setAttribute('deckdeckgo-prism', lang);
+      let language = definition.main ? definition.main : lang;
```

Comment:
@boradakash I have tried the  improvement with require but it doesn't work as here the extra languages  gonna be ignored.

for example

```
<deckgo-highlight-code language="t4">
      <code slot="code">Hello World</code>
    </deckgo-highlight-code>
```

should load `t4` and the require `t4-templating` and `csharp` but only `t4`, because of the `main` check, is going to be loaded

Reply:
I think `lang` should remain a paramater in this function, the might help